### PR TITLE
Introduce nativeOnMobile option and bugfix view that does not update on date mutation

### PR DESCRIPTION
**Pull request by strokyl**

### CODE REVIEW

File: lib/angular-clockpicker.js [hunk 1]
```diff
@@ -40,12 +39,27 @@
         };
       }
 
+      function parseMobileTime(string) {
+        if (!string) {
+          return;
+        } else if (string.match(/(AM|PM)\s*$/i)) {
+          return strictParse(true, string);
+        } else {
+          var withoutPotentialMinisecond = (string.trim().match(/^\d{1,2}:\d{1,2}/) || [null])[0];
```
Comment (Eriflow) [inline lib/angular-clockpicker.js]: what is Minisecond?
on opera when you do .val() on time fields it does return 12:03:05 with millisecond

File: README.md [hunk 2]
```diff
@@ -15,9 +15,17 @@ You can specify option of clock-picker documented [here](http://weareoutman.gith
...
 Changelog
 =========
 
 1.0.0
 
 * Renaming clockpicker-wrapper directive into lng-clockpicker
+
+1.1.0
+
+* Add nativeOnMobile options in order to delegate to system timepicker on mobile device
```
Comment (GrahamLinagora) [inline README.md]: on mobile devices ?

File: lib/angular-clockpicker.js (hunk 1 above)
Comment (strokyl) [inline lib/angular-clockpicker.js]: I replaced it with millisecond on my push -f

File: README.md (hunk 2 above)
Comment (strokyl) [inline README.md]: corrected

File: lib/angular-clockpicker.js [hunk 3]
```diff
@@ -57,22 +71,38 @@
...
 
       function getModelValue() {
         return ngModel.$modelValue ? ngModel.$modelValue.clone() : moment();
       }
 
-      getModelValue();
+      var parseViewValue = clockpickerService.parseTime.bind(null, isMobile && options.nativeOnMobile, options.twelvehour);
+
+      scope.$watch(function() {
+        return ngModel.$modelValue && ngModel.$modelValue && ngModel.$modelValue.unix && ngModel.$modelValue.unix();
```
Comment (Eriflow) [inline lib/angular-clockpicker.js]: There are two _ngModel.$modelValue_

**Comment by Eriflow** [comment]
:+1:

**Comment by Eriflow** [comment]
:+1:

**Pull request merged**